x-pack/plugins/maps/public/classes/styles/vector/vector_style.tsx

```diff
@@ -171,6 +172,154 @@ export class VectorStyle implements IVectorStyle {
     return getDefaultStaticProperties(mapColors);
   }
 
+  async _updateFieldsInDescriptor(
+    nextFields: IField[],
+    styleFieldsHelper: StyleFieldsHelper,
+    originalProperties: VectorStylePropertiesDescriptor,
+    previousFields: IField[],
+    mapColors: string[]
+  ) {
+    const invalidDynamicProperties = (Object.keys(originalProperties) as VECTOR_STYLES[]).filter(
```

Comment:
Can you rename `invalidDynamicProperties` to `invalidStyleNames`? Its confusing that `invalidDynamicProperties` is an array of `VECTOR_STYLES[]`. By the name, I would think `invalidDynamicProperties` contains `IDynamicStyleProperty[]` and that is not the case.